components/asset.rst

```diff
@@ -0,0 +1,251 @@
+.. index::
+   single: Asset
+   single: Components; Asset
+
+The Asset Component
+===================
+
+   The Asset component manages URL generation and versioning of web assets such
+   as CSS stylsheets, JavaScript files and image files.
+
+Installation
+------------
+
+You can install the component in two different ways:
+
+* :doc:`Install it via Composer </components/using_components>` (``symfony/asset`` on `Packagist`_);
+* Use the official Git repository (https://github.com/symfony/Asset).
+
+Usage
+-----
+
+Asset Packages
+~~~~~~~~~~~~~~
+
+The Asset component manages its assets through packages. A package groups all
+the assets which use the same versioning strategy. In the following basic
+example, a package is created to manage assets without any versioning:
+
+.. code-block:: php
+
+    use Symfony\Component\Asset\Package;
+    use Symfony\Component\Asset\VersionStrategy\EmptyVersionStrategy;
+
+    $package = new Package(new EmptyVersionStrategy());
+
+    echo $package->getUrl('/image.png');
+    // result: /image.png
+
+Packages implement the ``PackageInterface``, which defines the following two
+methods::
```

Comment:
I don't think we have to show the interface here, just a defintion list of the 2 methods and their usage.